src/audiomixcontroller/DefaultAudioMixController.ts

```diff
@@ -47,7 +55,7 @@ export default class DefaultAudioMixController implements AudioMixController {
      * Throw error if browser doesn't even support setSinkId
      * Read more: https://caniuse.com/?search=setSinkId
      */
-    if (!this.browserBehavior.supportsSetSinkId()) {
+    if (device && !this.browserBehavior.supportsSetSinkId()) {
```

Comment:
Question: If we do not do anything if it's a `null` device, then why do we even allow passing `null` device here in this function?

Reply:
In this function I think you might be right! These multiple entry points and shared logic are definitely DRY with lots of shared state, so it's hard to tell which steps can be elided. For example, `bindAudioMix` bails out if the old sink ID and new sink ID are the same, and I'm not sure it should!